content/en/docs/concepts/storage/volumes.md

```diff
@@ -142,37 +101,39 @@ spec:
   - name: test-volume
     # This AWS EBS volume must already exist.
     awsElasticBlockStore:
-      volumeID: <volume-id>
+      volumeID: "<volume id>"
       fsType: ext4
 ```
 
-#### CSI Migration
+#### AWS EBS CSI migration
 
 {{< feature-state for_k8s_version="v1.17" state="beta" >}}
 
-The CSI Migration feature for awsElasticBlockStore, when enabled, shims all plugin operations
+The `CSIMigration` feature for `awsElasticBlockStore`, when enabled, shims all plugin operations
```

Comment:
do you think "shims" is a clear word to describe this? or would something like "route" or "redirects" be more descriptive?

Reply:
No, not really. I think the docs could be improved here.

Reply:
I updated the use of "shim" in the page. PTAL.